index.html

```diff
@@ -26,8 +32,8 @@
 	<section class="bio">
 	<h1>Welcome to Shivani Sharma’s CSD120 Website!</h1>
 	<figure class="mugshot">
-	<a target="_blank" href="images/me.jpg" alt="me">
-		<img src="images/thumb.jpg" alt="thumb" width="304" height="228">
+	<a target="_blank" href="images/me.jpg">
+		<img src="images/thumb.jpg" alt="me" width="304" height="228">
```

Comment:
Good work on the figure element.  The alt attribute should only go on the `<img>` element